examples/gallery/src/text-layer.html

```diff
@@ -0,0 +1,123 @@
+<html>
+  <head>
+    <title>deck.gl TextLayer Example</title>
+
+    <script src="https://unpkg.com/deck.gl@^7.0.0/dist.min.js"></script>
+
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v0.50.0/mapbox-gl.js"></script>
+
+    <style type="text/css">
+      body {
+        width: 100vw;
+        height: 100vh;
+        margin: 0;
+      }
+      #tooltip:empty {
+        display: none;
+      }
+      #tooltip {
+        font-family: Helvetica, Arial, sans-serif;
+        position: absolute;
+        padding: 4px;
+        margin: 8px;
+        background: rgba(0, 0, 0, 0.8);
+        color: #fff;
+        max-width: 300px;
+        font-size: 10px;
+        z-index: 9;
+        pointer-events: none;
+      }
+      #control-panel {
```

Comment:
Could you apply similar styles of [website](http://deck.gl/#/documentation/deckgl-api-reference/layers/text-layer) demo for this `control-panel`?

For example, `box-shadow` could make the `control-panel` look better